sql/src/test/java/org/apache/druid/sql/calcite/CalciteArraysQueryTest.java

```diff
@@ -1014,8 +1015,14 @@ public void testArrayOverlapFilterDoubleArrayColumn()
   @Test
   public void testArrayOverlapFilterWithExtractionFn()
   {
+    final Map<String, Object> queryContext = QueryContexts.override(
+        QUERY_CONTEXT_DEFAULT,
+        ImmutableMap.of(PlannerContext.CTX_SQL_USE_EXTRACTION_FNS, true)
```

Comment:
nit: old habits die hard, but I think we've been starting to move towards using like `Map.of` and `List.of` instead of these